src/components/navbar/navbar.js

```diff
@@ -68,11 +68,6 @@ export default class Navbar extends React.PureComponent {
     }
   }
 
-  // TODO: Remove this on future releases
```

Comment:
I will not remove this until the nextmajor release (this is a breaking change for react < 16)

Reply:
fair enough